feature/security/gnsi/acctz/AccountingAuthenFailMulti/README.md

```diff
@@ -0,0 +1,52 @@
+# gNSI.acctz.v1 (Accounting) Test Accounting Authentication Failure - Multi-transaction
+
+Test Accounting for authentication failures of multi-transaction logins
+
+## Procedure
+
+- Record the current time, T0
+- For each of the possible RecordResponse.service_request.service_type that is not authenticated per-transaction:
```

Comment:
What's the definition of `authenticated per-transaction` v.s. `not authenticated per-transaction`? i.e. what's the difference between `AccountingAuthenFailMulti` and `AccountingAuthenFailUni`? Can you help give one example?